src/App.tsx

```diff
@@ -52,6 +52,20 @@ const UpgradeUndernames = React.lazy(
   () => import('./components/layout/UpgradeUndernames/UpgradeUndernames'),
 );
 
+const SettingsLayout = React.lazy(
```

Comment:
One thing that's strange is when you go to settings, then click between the panels, first time it's lazy loading and there's a visual glitch for loading that's a bit jarring. Would be nice to work this out in a way to avoid that. (Not a blocker for release.)